lib/common/resources/otel-collector-config.yml

```diff
@@ -55,6 +55,31 @@ spec:
                 regex: (.+)
                 target_label: __metrics_path__
                 replacement: /api/v1/nodes/$${1}/proxy/metrics/cadvisor
+            - job_name: 'apiserver'
```

Comment:
What is the challenge in adding a flag like blueprints. Since you have gone a larger hardship, i think we should add this with it unless you are running in to major challenge, pls explain